Assets/Scripts/Main.asmdef

```diff
@@ -2,7 +2,8 @@
     "name": "OperationBlackwell.Main",
     "rootNamespace": "OperationBlackwell",
     "references": [
-        "GUID:f039f22f2b49b14ccbd1ae39fcf00fdb"
+        "GUID:f039f22f2b49b14ccbd1ae39fcf00fdb",
```

Comment:
What are these references to?

Reply:
Textmesh pro would like a reference so I can change the text from scripts.

Reply:
And what is the second one then or is that just unity regen?

Reply:
The second one is likely the TextMeshPro one, this line change is just the added `,` as you can see, so unity regen.